Library/Homebrew/cmd/--repository.rb

```diff
@@ -2,10 +2,13 @@
 # frozen_string_literal: true
 
 require "abstract_command"
+require "shell_command"
 
 module Homebrew
   module Cmd
     class Repository < AbstractCommand
+      include ShellCommand
```

Comment:
@ZhongRuoyu What's the thoughts on why `include ShellCommand` was a better fit here than `class Repository < ShellCommand`?

Reply:
Haven't checked but the command system might detect ShellCommand as a command itself if it's a subclass of AbstractCommand.